pkg/space-manager/provider.go

```diff
@@ -188,6 +188,7 @@ func (p *provider) processProviderWatchEvents() {
 				chkErrAndReturn(logger, err, "Detected New space. Couldn't create the corresponding Space", "space name", spaceName)
 			} else {
 				// Space exists , just update its status
```

Comment:
This comment is no longer valid since we also update the config.  Might as well remove it if you can.

Reply:
Done